sqlitecluster/SQLite.cpp

```diff
@@ -165,7 +182,8 @@ void SQLite::commonConstructorInitialization() {
     }
 
     // WAL is what allows simultaneous read/writing.
-    SASSERT(!SQuery(_db, "enabling write ahead logging", "PRAGMA journal_mode = WAL;"));
+    string walMode = "wal"s + (wal2 ? "2" : "");
```

Comment:
it's to force it to a string literal from a `const char` so that the string concatenation works with everything after the `+`. https://en.cppreference.com/w/cpp/string/basic_string/operator%22%22s

Reply:
Is it me or it might be simpler just to deal with the single ternary, no concat.
```suggestion
    string walMode = wal2 ? "wal2" : "wal";
```